.github/workflows/ccpp.yml

```diff
@@ -26,6 +26,14 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Environment info
```

Comment:
This has no reason to exist anymore.

Reply:
It’s good for debug though, isn’t it? Don’t have to remember runner conf

Reply:
Sure.